pkg/client/src/app/pages/applications/analysis-wizard/review.tsx

```diff
@@ -10,28 +10,35 @@ import {
   TextContent,
   Title,
 } from "@patternfly/react-core";
-import { UseFormGetValues } from "react-hook-form";
+import { useFormContext } from "react-hook-form";
 
-import { IFormValues } from "./analysis-wizard";
 import { Application } from "@app/api/models";
+import { IReadFile } from "./components/add-custom-rules";
 
 interface IReview {
   applications: Application[];
-  getValues: UseFormGetValues<IFormValues>;
 }
 
-export const Review: React.FunctionComponent<IReview> = ({
-  applications,
-  getValues,
-}) => {
-  const {
-    mode,
-    targets,
-    withKnown,
-    includedPackages,
-    excludedPackages,
-    customRulesFiles,
-  } = getValues();
+const defaultScopes: Map<string, string> = new Map([
+  ["depsOnly", "Application and internal dependencies only"],
+  [
+    "depsAll",
+    "Application and all dependencies, including known Open Source libraries",
+  ],
+  ["depsSelect", "list of packages to be analyzed manually"],
+]);
+
+export const Review: React.FunctionComponent<IReview> = ({ applications }) => {
+  const { getValues } = useFormContext();
+  const mode: string = getValues("mode");
+  const targets: string[] = getValues("targets");
```

Comment:
Is there a centralized location where we can put these type definitions? 
```suggestion
  const { getValues, setValue } = useFormContext<IAnalysisWizardFormValues>();
  const { targets, sources } = getValues();```